modules/tide_landing_page/tide_landing_page.module

```diff
@@ -530,8 +530,9 @@ function _tide_landing_page_form_node_form_process(array $form, FormStateInterfa
       'callback' => 'tide_landing_page_set_value_hero_image_theme',
     ],
   ];
-    $header_style = \Drupal::state()->get($node->uuid() . '-' . 'header_style');
-    $form['_header_style']['_header_style_options']['#default_value'] = $node->isNew() ? 'default' : $header_style;
+    $header_style = \Drupal::state()->get($node->uuid() . '-header_style');
+    $form['_header_style']['_header_style_options']['#default_value'] = $header_style ?: 'default';
```

Comment:
the new node can be cloned node as well, so this check will make sure if there is any value set in drupal state that comes from the original to the new cloned node. If it does then it will move the original node value to the clone node as well. If it is empty then it will set it as default.